packages/lab/src/form-field/FormField.css

```diff
@@ -1,67 +1,45 @@
-/* Styles applied to the root element dependent on density */
-.uitk-density-touch {
-  --form-field-helper-text-margin-top: 8px;
-  --form-field-activation-indicator-icon-size: 7px;
-}
-.uitk-density-low {
-  --form-field-helper-text-margin-top: 4px;
-  --form-field-activation-indicator-icon-size: 5px;
-}
-.uitk-density-medium {
-  --form-field-helper-text-margin-top: 0px;
-  --form-field-activation-indicator-icon-size: 5px;
-}
-.uitk-density-high {
-  --form-field-helper-text-margin-top: 2px;
-  --form-field-activation-indicator-icon-size: 3px;
-}
-
-/* above, if `fillWidth={true}` */
-.uitk-density-touch {
-  --form-field-helper-text-margin-top: 5px;
+/* Style applied to the root element */
```

Comment:
This is an interesting one, I had to add :root here to allow for the variables to be found in :after pseudo.. there might be a better fix or perhaps im missing something @origami-z

Reply:
This will be a devtool bug, which works in Firefox.

However, i do wonder whether we should visual regression test on focus ring.

Reply:
good idea